app/src/main/java/ar/com/wolox/android/example/model/User.kt

```diff
@@ -0,0 +1,9 @@
+package ar.com.wolox.android.example.model
+
+import com.google.gson.annotations.SerializedName
+
+data class User(
+    @SerializedName("id") var id: Int,
+    @SerializedName("email")var email: String,
+    @SerializedName("password")var password: String
```

Comment:
If the name of the field is the same as the serializedName , you dont need to add de @SerializedName annotation. But it's a good practice anyway. Nice work